plugins/vpp/aclplugin/vppdump/dump_vppcalls.go

```
@@ -125,9 +128,9 @@ func DumpMACIPACL(swIfIndices ifaceidx.SwIfIndex, log logging.Logger, vppChannel
 	ruleMACIPData := make(map[ACLIdentifier][]*acl.AccessLists_Acl_Rule)
 
 	// get all ACLs with MACIP ruleData
-	MACIPRuleACLs, err := DumpMacIPAcls(log, vppChannel, stopwatch)
-	if err != nil {
-		return nil, err
+	MACIPRuleACLs, _ := DumpMacIPAcls(log, vppChannel, stopwatch)
```

Why ignore error here?